packages/ui/src/SelectField.tsx

```diff
@@ -24,6 +24,13 @@ const ClearIndicator = ({ innerProps }: IndicatorProps<SelectFieldOption<any>>)
   return <IconButton {...innerProps} icon={X} ariaHidden kind="primary" size="normal" className={styles.clear} tabIndex={-1} />
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const Input = (props: any) => {
+  // autoComplete='off' is hard-coded inside SelectField, but doesn't work in Chrome.
+  // Having an invalid value hard-coded disabled it in all browsers.
+  return <components.Input {...props} autoComplete="one-time-code" />
```

Comment:
Why not `chrome-off`?

Reply:
@aigoncharov I couldn't decide whether it's better to use an invalid a11y value or a value that shouldn't in practice effect functionality. I was 50/50, now I'm thinking an explicit chrome-off could be better, even though it's invalid.